napari/plugins/manager.py

```diff
@@ -0,0 +1,108 @@
+import importlib
+import os
+import pkgutil
+from logging import Logger
+
+import pluggy
+
+from . import _builtins, hookspecs
+
+logger = Logger(__name__)
+
+
+class NapariPluginManager(pluggy.PluginManager):
+    PLUGIN_ENTRYPOINT = "napari.plugin"
+    PLUGIN_PREFIX = "napari_"
```

Comment:
When I pip install my placeholder https://pypi.org/project/napari-segmentation/ plugin if I do `pip freeze` I see `napari-segmentation==0.0.0`

but when I run discover I get a count of 0.  I think the problem is with my very simple package https://github.com/sofroniewn/napari-segmentation as `import napari_segmentation` doesn't seem to work after pip install either .... this is probably something really basic with my setup.py, but was wondering if @tlambert03 you could point me in the right direction so I can make sure all this is working as expected

Reply:
there's nothing to import because you haven't added anything to `packages=[]` in your `setup.py`.  so the package at pypi is just metadata.

Reply:
Yup, that was it. If I `pip install napari-segmentation` (version `0.0.1`) this is now auto discovered. Thanks!!